src/components/Step/Step.js

```diff
@@ -14,7 +14,7 @@ export default class Step extends React.Component {
     /**
      * Content of the step
      */
-    children: PropTypes.node.isRequired,
+    children: PropTypes.node,
```

Comment:
if you can avoid passing this prop, than you should have or
1. Default children
2. Skip render `<Content />`

Reply:
skipped rendering content